tests/test_cases/test_force_release/test_force_release.py

```diff
@@ -190,3 +190,27 @@ async def test_force_followed_by_release_correct_value(dut):
     dut.stream_in_data.value = Release()
     await Timer(10, "ns")
     assert dut.stream_in_data.value == 0
+
+
+################################################################################
+# These two tests must be run back to back to ensure the behavior of writes
+# being applied by the beginning of the next test.
+
+
+@cocotb.test
+async def test_apply_force(dut) -> None:
+    """Release a Force as the last operation in a test."""
+    dut.stream_out_data_comb.value = Force(42)
+    await Timer(10, "ns")
+    dut.stream_out_data_comb.value = Release()
+
+
+@cocotb.test
```

Comment:
```suggestion
@cocotb.test(expect_fail=True)
```

Reply:
We could probably get this in now with this change. This will start failing after we fix the issue and it starts passing. It looks like NVC and Verilator work anyway, so the expect_fail expression would need to check for those sims.